### ISSUE TITLE
Update for US

### ISSUE THREAD

**Issue by eogitbot** [w3c/wai-policies-prototype#326]
Update for US

Submitted by [Judy Brewer](mailto:jbrewer@w3.org):

> US Section 508 was updated over a year ago to reference WCAG 2.0 rather than WCAG 1.0
> 
> Please list the US Access Board as the primary agency responsible for Section 508, though GSA also has an info site on 508 that should be listed.

~~~yaml
---
country:
  en: US
  # Manual enter other country names: 
updated: 2018-02-08
updatemsg:
# Related page:
province: 
policies:
  - title:
      en: Section 508
    url: https://www.access-board.gov/
    updated: 2017
    wcagver: WCAG 2.0, 
    enactdate: 2017
    type: Procurement law, 
    ministries:
      - title:
          en: "Most US federal agencies"
        url:
          en: ""
    webonly: false
    scope: 
    standard:
      - title:
          en: ""
        desc: ""
        url:
          en: ""
    documents:
      - title:
          en: ""
        desc: ""
        url:
          en: ""
---
~~~

**Comment by yatil** [comment]
@iamjolly, @maryjom Can you take a look at this update? Thanks!

**Comment by iamjolly** [comment]
Yes, we need to update this to have the new Section 508 info listed. We can remove references to WCAG 1.0 and the mention of "Updated standard now available but not enforced yet, see Revised section 508 standards below.")

Thanks @yatil and @brewerj.

**Comment by maryjom** [comment]
Actually, the enforced 508 is still the original one.

**Comment by slhenry** [comment]
/me assigned to Sharron & Brent so they can help manage getting this addressed :-)

**Comment by brewerj** [comment]
Hi Mary Jo, I don't believe so; the new policy went into effect on the one-year anniversary of the Refresh Update. And in any case, the Access Board had been advising everyone to use WCAG 2.0 by equivalent facilitation for about 3 years before that. Would be great to get this updated, after all the work on the Refresh. Thanks. - Judy

**Comment by iamjolly** [comment]
I'm a bit lost on what to update on this page based on the thread here. I'm happy to make whatever updates are needed, but I want to ensure what's needed is clear to me.

This seems like a special case where the law is using the updated/refreshed/harmonized guidelines with WCAG 2.0 AA, but an agency related to defining procurement requirements is holding back compliance by the affected entities. Is that an accurate summary of the comments so far?

If Section 508 is still the applicable law, and that law has been updated, then should we indicate, somehow, that the old 508 (WCAG 1.0-based) law is, in essence, deprecated and that folks need to follow the new law even if procurement guidelines have not been issued yet?

**Comment by slhenry** [comment]
@brewerj , please confirm you're OK with the changes (DIFF is https://github.com/w3c/wai-policies-prototype/pull/332), and if so, close this issue. Thanks.

**Comment by maryjom** [comment]
This was published in March.  Must have missed linking it to the pull request.
